res/windows/wix/RemoveConfigsAndLogs.bat

```diff
@@ -0,0 +1,6 @@
+@echo off
+:UseChoice
+%SystemRoot%\System32\choice.exe /C YN /N /M "Delele configuration and logs [Y/N]?"
+if not errorlevel 1 goto UseChoice
```

Comment:
Does this mean it will not exit unless user agrees to wipe everything?

Reply:
:) No, it doesn't. This is a workaround for the case when a user presses "Ctrl+C/Break" instead of "Y" or "N".

Reply:
Hm, I'm not familiar with syntax, but the way I read the docs and code, it loops back to `UseChoice` unless user typed `Y`. So they will be stuck in a loop until they agree.

Reply:
You can just test it yourself! :) Batch scripts are quite tricky to write.